Sources/TuistCore/ValueGraph/ValueGraphTraverser.swift

```diff
@@ -19,18 +19,51 @@ public class ValueGraphTraverser: GraphTraversing {
         self.graph = graph
     }
 
+    public var hasRemotePackages: Bool {
+        graph.packages.values.flatMap(\.values).first(where: {
```

Comment:
I think this is better described with `contains(where: {...})` than `first(where: {}) != nil`